cfme/cloud/provider/ec2.py

```diff
@@ -51,6 +54,18 @@ class EC2InstanceUtilizationView(VMUtilizationView):
     vm_network = LineChart(id='miq_chart_parent_candu_2')
 
 
+class EC2AzoneUtilizationView(AzoneCloudUtilizationView):
+    """Availability zone Utilization view for AWS providers"""
+    azone_disk = LineChart(id='miq_chart_parent_candu_1')
+    azone_network = LineChart(id='miq_chart_parent_candu_2')
+    azone_network_avg = LineChart(id='miq_chart_parent_candu_2_2')
+    azone_instance = LineChart(id='miq_chart_parent_candu_4')
```

Comment:
Required:
I think; you need to remove this as you added `VersionPicker`.

Reply:
Yes, absolutely.Overlooked that one.

That needs to go because of the VersionPicker.